stacks/spica/src/passport/interfaces/api-key.ts

```diff
@@ -0,0 +1,27 @@
+import {Policy} from "./policy";
+import {Observable} from "rxjs";
+import {IndexResult} from "@spica-server/core";
+import {HttpClient} from "@angular/common/http";
+
+export interface ApiKey {
+  _id?: string;
+  key?: string;
+  name: string;
+  description?: string;
+  policies?: Policy[];
```

Comment:
Type of this field should be `string[]`

Reply:
I couldn't understand why we use **string[]** instead of **Policy[]**.

Reply:
cause we just store ids of policies in there.

Reply:
So we won't need to update api key when something changed on policy. Is that why we store id ?

Reply:
Nope. we just store the id cause the original policy is stored somewhere else. [Relation]